src/arcaflow_plugin_sdk/schema.py

```diff
@@ -1,20 +1,94 @@
+"""
+This module provides the tools to build a schema, either by hand, or automatically from type hints and annotations.
+
+Using
+=====
+
+For using this module please check the documentation located at
+https://arcalot.github.io/arcaflow/creating-plugins/python/
+
+Contributing
+============
+
+This module is structured by using region folding delimiters (e.g. # region SomeRegion). You can use an editor that
+supports these regions to collapse them together for easier access. The module has the following regions:
+
+Exceptions
+----------
+
+This region defines a number of exceptions that can be raised either while building a schema or while working with
+input and output.
+
+Annotations
+-----------
+
+Annotations can be applied to dataclass fields to convey extra information (e.g. minimum or maximum values). These
+annotations are used when building a schema. Note, that unlike tools like Pydantic we do not validate the dataclass
```

Comment:
"Note, that unlike tools like Pydantic we do not validate the dataclass" -> "Note that, unlike tools like Pydantic, we do not validate the dataclass"